README.md

```diff
@@ -1 +1,178 @@
-# toDoList
```

Comment:
![image](https://github.com/oluwatobi-fasasi/toDoList/assets/93039359/ab408905-5b0f-4e01-af53-38c8d5d292b0)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional and descriptive 😊